<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TilinValintaDialogi</class>
 <widget class="QDialog" name="TilinValintaDialogi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Valitse tili</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="view"/>
   </item>
   <item>
    <widget class="QLineEdit" name="suodatusEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="suosikitNappi">
       <property name="text">
        <string>Suosikit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="valitseNappi">
       <property name="text">
        <string>Valitse</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="peruNappi">
       <property name="text">
        <string>Peru</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>valitseNappi</sender>
   <signal>clicked()</signal>
   <receiver>TilinValintaDialogi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>peruNappi</sender>
   <signal>clicked()</signal>
   <receiver>TilinValintaDialogi</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
